app/assets/javascripts/eventscripts/create_event.js

```diff
@@ -58,5 +59,29 @@ $( document ).on('turbolinks:load', function() {
            $("#event_enddate").val(formattedDate);
        }
     });
+
+    // Autofill of player count for an event
+    $("#event_player_type").on("change", showPlayerCount);
+    $("#event_min_players_per_team").hide();
```

Comment:
Why do you set the player count here too? It is set in line 64 anyways... This might cause the error you were having with the fields not showing up anymore if you reload the site (because you hide it at the beginning anyways at every time).